src/shaders/line.fragment.glsl

```diff
@@ -50,11 +45,8 @@ void main() {
     float blur2 = (blur + 1.0 / u_device_pixel_ratio) * v_gamma_scale;
     float alpha = clamp(min(dist - (v_width2.t - blur2), v_width2.s - dist) / blur2, 0.0, 1.0);
 #ifdef RENDER_LINE_DASH
-    float sdfdist_a = texture2D(u_dash_image, v_tex_a).a;
-    float sdfdist_b = texture2D(u_dash_image, v_tex_b).a;
-    float sdfdist = mix(sdfdist_a, sdfdist_b, u_mix);
-    float sdfwidth = min(dash_from.z * u_scale.y, dash_to.z * u_scale.z);
-    float sdfgamma = 1.0 / (2.0 * u_device_pixel_ratio) / sdfwidth;
+    float sdfdist = texture2D(u_dash_image, v_tex).a;
+    float sdfgamma = 1.5 / (2.0 * u_device_pixel_ratio) / dash_to.z;
```

Comment:
I think we should change the 1.5 to 1.0 here. If I'm reading it correctly, that would make it fade over 1 pixel. When trying it out it looks slightly sharper (in a good way).

It should be fine to update all the affected render tests. The current slightly-fuzzy behavior was probably a bug.

Reply:
Maybe just one caveat is that the derivative of smoothstep is greater than 1 at x=0.5, so that smoothstep over width 1.5 might actually be comparable to linearstep over 1px, just with a bit smoother falloff. Which is to say, smoothstep over 1px might end up a bit jagged. I'm not really sure what's best, but I've been using linearstep in place of smoothstep lately. 🤷 

```glsl
float linearstep(float edge0, float edge1, float x) {
    return  clamp((x - edge0) / (edge1 - edge0), 0.0, 1.0);
}
```

smoothstep with dy/dx=1.5 at x=0.5:

<img width="491" alt="Screen Shot 2022-10-26 at 9 58 25 AM" src="https://user-images.githubusercontent.com/572717/198089031-8531ca72-983a-4b4b-8b66-7f4174872dc3.png">

Reply:
@rreusser thanks so much for the suggestion! Added this in and it seems to work great.